src/m365/spo/commands/tenant/tenant-commandset-get.ts

```diff
@@ -114,7 +116,8 @@ class SpoTenantCommandSetGetCommand extends SpoCommand {
 
       if (listItemInstances?.value.length > 0) {
         if (listItemInstances.value.length > 1) {
-          throw `Multiple ListView Command Sets with ${args.options.title || args.options.clientSideComponentId} were found. Please disambiguate (IDs): ${listItemInstances.value.map(item => item.Id).join(', ')}`;
+          const resultAsKeyValuePair = formatting.convertArrayToHashTable('Id', listItemInstances.value);
+          listItemInstances.value[0] = await Cli.handleMultipleResultsFound<ListItemInstance>(`Multiple ListView Command Sets with ${args.options.title || args.options.clientSideComponentId} were found.`, resultAsKeyValuePair);
```

Comment:
Another one to slightly refactor. Could you Make sure the deletion of the ID is carried out?

Reply:
yes of course. Just wondering if we relay here on `Id` not `ID` why would you find this as required?